src/main/java/http/utils/MainArgumentParser.java

```diff
@@ -0,0 +1,35 @@
+package http.utils;
+
+public class MainArgumentParser {
+    private int portNumber;
+    private String workingDirectory;
+
+    public MainArgumentParser(String[] args) {
+        portNumber = 5000;
+        workingDirectory = "./";
+
+        for (int i = 0; i < args.length; i++) {
+            try {
+
+                if (args[i].equals("-p")) {
+                    portNumber = Integer.parseInt(args[i]);
+                }
+                if (args[i].equals("-d")) {
```

Comment:
If we wanted to have 8 different argument flags, would we end up with 8 `if` statements??